packages/gatsby-theme-newrelic/src/components/SearchInput.js

```diff
@@ -97,18 +98,18 @@ const SearchInput = forwardRef(
           }}
           css={css`
             width: 100%;
-            border: 1px solid var(--border-color);
+            border: 0px solid var(--border-color);
```

Comment:
```suggestion
            border: 1px solid transparent;
```

I see that a border is added on hover. By reducing this to `0`, I fear this will make the surrounding UI jump since it goes from no border to a 1px border around it. Rather than removing the border in the default state, instead let's change the border color to transparent so that it takes up the same amount of room and avoids jumping around.

Reply:
@zstix and I were trying to figure out where this was occurring. Thanks for that!